central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -370,15 +381,15 @@ func (m *managerImpl) HandleResult(sensorCtx context.Context, result *storage.Co
 	if err != nil {
 		if errors.Is(err, watcher.ErrComplianceOperatorReceivedOldCheckResult) {
 			log.Debugf("The CheckResult is older than the current scan in the store")
-			return nil
+			return err
 		}
 		if errors.Is(err, watcher.ErrComplianceOperatorScanMissingLastStartedFiled) {
 			log.Debugf("The scan is missing the LastStartedField: %v", err)
```

Comment:
```suggestion
			log.Debugf("The scan is missing the LastStartedTime field: %v", err)
```
but the sentinel error says the same (`scan is missing the LastStartedTime field`) so this may look strange as:

`The scan is missing the LastStartedField: scan is missing the LastStartedTime field`

Reply:
Thanks! There is still the `LastStartedTime` vs `LastStartedField` and that is in 2 places. Maybe search globally the code `LastStartedField` and double-check whether that should not be `LastStartedTime`? (I think, it should)